S7.Net/Conversion.cs

```diff
@@ -138,19 +138,45 @@ public static string ValToBinString(this object value)
 
         /// <summary>
         /// Helper to get a bit value given a byte and the bit index.
-        /// Example: DB1.DBX0.5 -> var bytes = ReadBytes(DB1.DBW0); bool bit = bytes[0].SelectBit(5); 
+        /// Example: DB1.DBX0.5 -> var bytes = ReadBytes(DB1.DBW0); bool bit = bytes[0].SelectBit(5);
         /// </summary>
-        /// <param name="data"></param>
-        /// <param name="bitPosition"></param>
-        /// <returns></returns>
-        public static bool SelectBit(this byte data, int bitPosition)
+        /// <param name="data">byte data get from</param>
+        /// <param name="index">bit index</param>
+        /// <returns>bool value will get</returns>
+        public static bool SelectBit(this byte data, int index)
         {
-            int mask = 1 << bitPosition;
+            int mask = 1 << index;
             int result = data & mask;
 
             return (result != 0);
         }
 
+        /// <summary>
+        /// Helper to set a bit value to the given byte at the bit index.
+        /// Example: byte data = (byte)0x00; data.SetBit(4, true);// data -> 0x10
```

Comment:
The summary should just be that, the summary. The example can be moved into a separate block:

```csharp
  <example>
      Set the bit at index 4:
      <code>
        byte data = 0;
        data.SetBit(4, true);
      </code>
  </example>
```